wrappers/react/test/_helpers.tsx

```diff
@@ -32,12 +32,30 @@ export function mockElementDimensions(element, width, height) {
 }
 
 export function simulateKeyboardEvent(type, keyCode) {
-  const event = document.createEvent('KeyboardEvent');
-  const init = (event as any).initKeyboardEvent !== void 0 ? 'initKeyboardEvent' : 'initKeyEvent';
+  const newEvent = document.createEvent('KeyboardEvent');
+  // const init = (event as any).initKeyboardEvent !== void 0 ? 'initKeyboardEvent' : 'initKeyEvent';
 
-  event[init](type, true, true, window, false, false, false, false, keyCode, 0);
+  // Chromium Hack
+  Object.defineProperty(newEvent, 'keyCode', {
+    get : function() {
+        return this.keyCodeVal;
+    }
+  });     
+  Object.defineProperty(newEvent, 'which', {
```

Comment:
Both `keyCode` and `which` are deprecated. Shouldn't we rather remove them from our code?

Reply:
Unfortunately, we use `keyCode` internally in many places:
- editorManager,
- walkontable.overlays,
- autocompleteEditor,
- dateEditor,
- handsontableEditor,
- selectEditor,
- textEditor, ​
- contextMenu.menu,
- filters (component/ui),
- mergeCells,
- undoRedo,
- checkboxRenderer,

I adjusted creating keyboard events accordingly to their specification (some differences in arguments order etc.). Initially, I was hopeful that we could use `new KeyboardEvent()`- however - it's not supported by IE (I assume we run wrappers' tests on IE).